Update SHA in whitelist for file used by Vendor 2274

andfinally opened the pull request:
In https://github.com/Automattic/php-malware-finder/pull/16, we seem to have used the wrong SHA, because this upload is still failing the scan. I generated this new SHA with

```
yara -r ./php.yar ~/Downloads/[ plugin ]/lib/receipt/dist/receipt.iife.js | sed -e 's/.* //' | xargs shasum | uniq
```

## Test instructions

1. Have `yara` installed (e.g. `brew install yara`)
2. Download the affected JS file.
3. In the `php-malware-finder` subdirectory, run `yara -sr ./php.yar /path/to/the/file.js`; observe failures beginning with `$nano: $1...`
4. Switch to this branch
5. Run the test again and see that the failures have disappeared

Ignore the message `warning: rule "DodgyPhp" in ./php.yar(93): $pr contains .*, .+ or .{x,} consider using .{,N}, .{1,N} or {x,N} with a reasonable value for N` and not any `DodgyStrings` or `ObfuscatedPhp` errors.

Dan-Q reviewed (approved):
I believe the checksum is incorrect (but the test instructions are wrong: they point to an unminified version of the JS file).

andfinally commented:
Thanks Dan, removed the outdated link from the test steps. That was the previous version of the file. As we're finding, this file has changed since we added it to the whitelist.
